client/src/helpers/translate/localeEN-NL.js

```diff
@@ -40,5 +40,17 @@ export default {
   results: {
     [NL]: "Resultaten",
     [EN]: "Results"
+  },
+  aboutPageText: {

```

Comment:
Kleine feedback op de content:
- Die beginzin is een beetje gek geformuleerd naar mijn mening... ik zou in ieder geval "zo blij dat je er bent" weghalen of anders schrijven.
- Het gaat hier specifiek over HBO-ICT minors.
- Ik heb een naam en die is Maurits :P